data/main.ui

```diff
@@ -40,7 +40,7 @@
                       <object class="GtkButton" id="reset-button">
                         <property name="valign">start</property>
                         <property name="visible">True</property>
-                        <property name="label">Reset</property>
+                        <property name="label" translatable="yes" context="notebook container">Reset</property>
```

Comment:
What does `context="notebook container"` mean here and why is it specified for all the translatable strings?